.github/workflows/generate-provider-docs.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       - name: Install Hugo
         uses: peaceiris/actions-hugo@v2
         with:
-          hugo-version: '0.96.0'
+          hugo-version: '0.111.0'
```

Comment:
will we  need to update the version of hugo we are running locally in order to build the site locally going forward as well? I'm guessing this new version is what is allowing us to add JSON as an output type?

Reply:
Only if you want to build pulumi/docs locally; pulumi-hugo and others don’t need it — although we should probably upgrade everywhere (just as a rule) so everything’s on the same version. I’m happy to to that.

This PR uses the newer version specifically for the `Ancestors` property, though, just so you’re aware. We use this property in the JSON output script, but JSON output doesn’t require it specifically.